lib/storage.js

```diff
@@ -50,7 +50,8 @@ Storage.prototype._createIndexes = function() {
     id: 1,
   });
   this.db.collection(collections.ADDRESSES).createIndex({
-    walletId: 1
+    walletId: 1,
+    createdOn: 1,
```

Comment:
lol, I love to be able to just end with a comma, I don't have to change two lines if I need to add another field :smile: